SU-HSDO/suhumsci
# Arch 71 publications list page

Changes to Publications view config, Twig & Sass for masonry and masonry item patterns.
# READY FOR REVIEW
- I still need to update the patterns in the style guide, but would like code review first.

# Summary
- Changes to Publications view config, Twig & Sass for masonry and masonry item patterns.

# Needed By (Date)
- End of current sprint.

# Urgency
- Pretty urgent, if we want the publications page to be styled.

# Steps to Test
1. Update your environment's Publications module config (/admin/config/development/configuration/report/module/hs_publications) -- import all items
1. Create some Publications nodes (consider using devel generate, though author fields are very long)
1. Create a basic page for Publications, and add the Publications view display to it.
1. Visit the new Publications node and confirm behavior and appearance are satisfactory.

# Affected Projects or Products
- None.

# Associated Issues and/or People
- ARCH-71
